lms/static/sass/discussion/views/_response.scss

```diff
@@ -64,40 +62,14 @@
 
 // +element - add response area
 .container .discussion-body .add-response {
-  margin-top: $baseline;
-  padding: 0 ($baseline*1.5);
-
-  .add-response-btn {
-    @include white-button;
-    @include linear-gradient(top, $white 35%, $gray-l4);
-    position: relative;
-    border: 1px solid $gray-l2;
-    padding: 0 18px;
-    width: 100%;
-    box-shadow: 0 1px 1px $shadow-l1;
-    @include text-align(left);
-    font-size: 13px;
-
-    .fa-reply:before { // flip the icon for RTL
-      @include ltr {
-        content: "\f112"; // FA icon arrow to the left
-      }
-
-      @include rtl {
-        content: "\f064"; // FA icon arrow to the right
-      }
-    }
-
-    span.add-response-btn-text {
-      @include padding-left($baseline/5);
-    }
-  }
+  display: inline;
+  padding: $baseline/2;
 }
 
 // +CASE: answered question - collapsed comments in answers
 .forum-response .action-show-comments {
-  @include box-sizing(border-box);
-  @include font-size(13);
+  box-sizing: border-box;
+  font-size: 13px;
```

Comment:
We should use one of our defined size variables, or use rem/em. Also 13px is tiny. 16px should be our minimum. If we absolutely need smaller, 14px.